Expensify/App
# Add new action page for payment methods

cc: @marcaaron 
### Details
This PR just adds some data to ONYX which will be used in other PRs in this tracking [issue](https://github.com/Expensify/Expensify/issues/154278). We are doing this first without other code to help parallelize work.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/166271

### Tests
1. Import the function in the payments page
2. Go to the payments page
3. Check onyx and make sure that the user's wallet, cards, and bank accounts are saved

### QA Steps
None

### Tested On
